src/main/msp/msp_serial.h

```diff
@@ -27,7 +27,9 @@
 #include "msp/msp.h"
 
 // Each MSP port requires state and a receive buffer, revisit this default if someone needs more than 3 MSP ports.
-#define MAX_MSP_PORT_COUNT 4
+#ifndef MAX_MSP_PORT_COUNT 
+#define MAX_MSP_PORT_COUNT 6  
```

Comment:
please revert `4`. The idea is that the code default remain 4 and pilot may define pilot's choice at build-time.